src/vmc/assets/documents.py

```diff
@@ -143,7 +153,11 @@ def get_assets_with_tag(tag: str, config=None):
     def update_gone_discovered_assets(targets, scanned_hosts, discovered_assets, config=None):
         index = AssetDocument.get_index(config)
         # FIXME: update by query
```

Comment:
should that comment stay?

Reply:
Yes, I don't have idea how to make this now